releases/note-source/v15_0.rst

```diff
@@ -0,0 +1,239 @@
+Spring 2018 Release (v15_0)
+===========================
+
+.. ATTENTION::
+   This will be the last major release that supports Python2. From release v16.0 (Fall 2018) on, only Python3 be explicitly supported. Minor releases of v15 will continue to support Python2.
+
++-------------------------------------------+------------+
+| Source                                    | Identifier |
++===========================================+============+
+| Git tag                                   | 15.0       |
++-------------------------------------------+------------+
+| :doc:`EUPS distrib </install/newinstall>` | v15\_0     |
++-------------------------------------------+------------+
+
+- :ref:`release-v15-0-major-changes`
+- :ref:`release-v15-0-bug-fixes`
+- :ref:`release-v15-0-internal-improvements`
+
+.. - :ref:`release-v15-0-sui`
+.. - :ref:`Qserv and Data Access <release-v15-0-qserv-dax>`
+
+*See also:*
+
+- :doc:`Installation instructions <../install/index>`
+- :doc:`Known issues </known-issues>`
+- `Characterization Metric Report (DMTR-62) <https://ls.st/DMTR-62>`_
+
+These release notes highlight significant changes to the Science Pipelines codebase which are likely to be of wide interest.
+For a complete list of changes, see :ref:`release-v15-0-tickets`.
+
+.. _release-v15-0-major-changes:
+
+Major Functionality and Interface Changes
+-----------------------------------------
+- :ref:`release-v15-0-diffim`
+- :ref:`release-v15-0-mem`
+- :ref:`release-v15-0-artifacts`
+- :ref:`release-v15-0-skysub`
+- :ref:`release-v15-0-wcs`
+- :ref:`release-v15-0-firefly`
+- :ref:`release-v15-0-compression`
+- :ref:`release-v15-0-cmdline`
+- :ref:`release-v15-0-prereqs`
+
+
+
+.. _release-v15-0-diffim:
+
+Image differencing algorithm improvements
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Algorithms appropriate for differencing in a variety of contexts have been implemented as tasks in the LSST stack. These various algorithms and configurations have been compared.  The results are written up in `DMTN-061 <https://dmtn-061.lsst.io>`_.
+
+.. _release-v15-0-mem:
+
+Performance improvements in coaddition
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Now all coaddition algorithms have significantly reduced memory footprints
+
+.. _release-v15-0-artifacts:
+
+Significantly improved artifact rejection in coaddition
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Coaddition algorithms that do artifact clipping can now handle artifacts that overlap from epoch to epoch.
+
+.. _release-v15-0-skysub:
+
+Full focal plane sky subtraction
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+There are now tasks to create and apply models of the sky that extend over the entire field of view. View notes on `CLO <https://community.lsst.org/t/sky-subtraction/2415>`_.
+
+.. _release-v15-0-wcs:
+
+Replace all ``Wcs`` classes with the `AST <https://arxiv.org/abs/1602.06681>`_ backed ``SkyWcs``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
Underline the wrong length?

Reply:
Also, I'd generally avoid putting a link in a header (even if it technically works)

Reply:
Tim asked me to do that, but I guess I can link the first mention of AST in the body.